sources/packages/api/src/route-controllers/institution-user/institution-user.controller.service.ts

```diff
@@ -0,0 +1,369 @@
+import {
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import {
+  BCeIDService,
+  InstitutionService,
+  InstitutionUserAuthService,
+  UserService,
+} from "../../services";
+import {
+  PaginationOptions,
+  PaginatedResults,
+  CustomNamedError,
+  getUserFullName,
+} from "../../utilities";
+import { InstitutionUser } from "../../database/entities";
+import {
+  CreateInstitutionUserAPIInDTO,
+  InstitutionUserAPIOutDTO,
+  UpdateInstitutionUserAPIInDTO,
+  UserActiveStatusAPIInDTO,
+} from "./models/institution-user.dto";
+import { PrimaryIdentifierAPIOutDTO } from "../models/primary.identifier.dto";
+import { BCeIDAccountTypeCodes } from "../../services/bceid/bceid.models";
+import { ApiProcessError } from "../../types";
+import {
+  BCEID_ACCOUNT_NOT_FOUND,
+  INSTITUTION_MUST_HAVE_AN_ADMIN,
+  INSTITUTION_USER_ALREADY_EXISTS,
+  LEGAL_SIGNING_AUTHORITY_EXIST,
+} from "../../constants";
+import { InstitutionUserTypes } from "../../auth/user-types.enum";
+
+/**
+ * Service/Provider for Institutions controller to wrap the common methods.
+ */
+@Injectable()
+export class InstitutionUserControllerService {
+  constructor(
+    private readonly institutionService: InstitutionService,
+    private readonly bceIDService: BCeIDService,
+    private readonly userService: UserService,
+    private readonly institutionUserAuthService: InstitutionUserAuthService,
+  ) {}
+
+  /**
+   * Get institution users with page, sort and search.
+   * @param institutionId
+   * @param paginationOptions
+   * @returns Institution Users.
+   */
+  async getInstitutionUsers(
+    institutionId: number,
+    paginationOptions: PaginationOptions,
+  ): Promise<PaginatedResults<InstitutionUserAPIOutDTO>> {
+    const [institutionUsers, count] =
+      await this.institutionService.getInstitutionUsers(
+        institutionId,
+        paginationOptions,
+      );
+
+    return {
+      results: institutionUsers.map((institutionUser: InstitutionUser) => {
```

Comment:
to simply ` results: institutionUsers.map((institutionUser: InstitutionUser) => 
        this.transformToInstitutionUserAPIOutDTO(institutionUser)
      }),`

Reply:
Changed!